doc/SdA.txt

```diff
@@ -177,14 +177,14 @@ The user can run the code by calling:
   python code/SdA.py
 
 By default the code runs 15 pre-training epochs for each layer, with a batch
-size of 1. The corruption levels are 0.1 for the first layer, 0.2 for the second,
+size of 20. The corruption levels are 0.1 for the first layer, 0.2 for the second,
 and 0.3 for the third. The pretraining learning rate is 0.001 and 
-the finetuning learning rate is 0.1. Pre-training takes 585.01 minutes, with 
-an average of 13 minutes per epoch. Fine-tuning is completed after 36 epochs
-in 444.2 minutes, with an average of 12.34 minutes per epoch. The final 
-validation score is 1.39% with a testing score of 1.3%. 
-These results were obtained on a machine with an Intel
-Xeon E5430 @ 2.66GHz CPU, with a single-threaded GotoBLAS.
+the finetuning learning rate is 0.1. Pre-training takes 71.43 minutes, with
+an average of 1.6 minutes per epoch. Fine-tuning is completed after 243 epochs
+in 441.3 minutes, with an average of 1.9 minutes per epoch. The final
+validation score is 1.7% with a testing score of 1.6%.
```

Comment:
These results are quite bad, actually, and I don't think that is what we should advertise.
This is on par with non-pretrained networks, and it is worse than pre-training with regular autoencoders (1.4% / 1.4%), whereas denoising autoencoder usually outperform regular ones.
The number of pre-training epochs, as well as the pre-training and fine-tuning learning rates should probably be updated after changing the batch size.